pkg/util/util.go

```diff
@@ -124,10 +131,16 @@ func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
 			defer HandleCrash()
 			f()
 		}()
+
+		jitteredPeriod := period
+		if jitterFactor > 0.0 {
```

Comment:
I could repeat the TestUntil as TestJitterUntil for non zero jitter factor and check the function is never called when a channel is closed before the loop. The same for TestUntilReturnsImmediately.

For jitter specific behaviour, I could check one loop of the Until does not take more then duration + maxFactor * duration + some small number covering overhead. Then to test for negative number that one loop takes duration + some small number covering overhead. The function can be empty function for this scenario.